### ISSUE TITLE
Different dating

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#6]
Line 40: ADT[@etichetta="Different dating"] other dating: when this field is used, does that mean that there may be uncertainty about the date of creation of the photo? Would you not model it as another potential creation date for the photo?

**Comment by marilenadaquino** [comment]
Yes, to be precise another E12_Production, because ADT refers to the date of the print (see issue 7).
